### PR TITLE
feat: order detail payment section fixed

### PR DESCRIPTION
**Pull request by nehasacher143**
**Task:** Fix the order detail page for Authorizenet.

**Screenshot:
Before:**
screenshot-localhost_8001-2020 05 04-17_37_24

**After:**
screenshot-localhost_8001-2020 05 04-17_36_35

### CODE REVIEW

File: erpnext/templates/pages/order.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {% extends "templates/web.html" %}
 {% from "erpnext/templates/includes/order/order_macros.html" import item_name_and_description %}
 
+<!--  -->
```
Comment (Alchez) [inline erpnext/templates/pages/order.html]: Remove this line

File: erpnext/templates/pages/order.html [hunk 2]
```diff
@@ -157,15 +159,15 @@ <h1 class="m-0">{{ doc.name }}</h1>
...
-					</div>
 				</div>
+			</div>
+								<div class="page-header-actions-block" data-html-block="header-actions">
+									<p class="text-right">
+										<a href="#" onclick="initiate_payment()" class="btn btn-primary btn-sm" id="pay-for-order">{{ _("Pay") }}
+											{{ doc.get_formatted("grand_total") }}
+										</a>
+									</p>
+								</div>
```
Comment (Alchez) [inline erpnext/templates/pages/order.html]: Fix the indentation for this block

**Review by Alchez — CHANGES_REQUESTED** [review]

**Comment by nehasacher143** [comment]
@Alchez Please review

File: erpnext/templates/pages/order.html [hunk 3]
```diff
@@ -118,90 +114,94 @@ <h1 class="m-0">{{ doc.name }}</h1>
...
 
 {% endblock %}
 
 {% block script %}
-<script> {% include "templates/pages/order.js" %} </script>
+<script>
+	{
+		% include "templates/pages/order.js" %
+	}
+</script>
```
Comment (Alchez) [inline erpnext/templates/pages/order.html]: This threw a syntaxError the last time, revert this one

**Review by Alchez — APPROVED** [review]

**Review by Alchez — APPROVED** [review]

**Comment by Alchez** [comment]
Updated to existing payment gateway [upstream PR](https://github.com/frappe/erpnext/pull/21374).

**Pull request merged**